<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-GGQ96VIC" failures="1" tests="5" name="Test" time="219.084" errors="0" timestamp="2022-11-11T16:11:53 IST">
  <testcase classname="features.Login" name="ValidTest" time="22.363"/>
  <testcase classname="features.Logout" name="LogoutTest" time="25.521">
    <failure type="java.lang.AssertionError" message="expected [Logou successful] but found [Logout successful]">
      <![CDATA[java.lang.AssertionError: expected [Logou successful] but found [Logout successful]
at features.Logout.LogoutTest(Logout.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- LogoutTest -->
  <testcase classname="features.TrackOrder" name="TrackOrderTest" time="14.459"/>
  <testcase classname="features.Checkout" name="CheckoutTest" time="46.67"/>
  <testcase classname="features.ContactForm" name="ContactFormTest" time="40.298"/>
</testsuite> <!-- Test -->
